Backpack-SwiftUI/Badge/Classes/BPKBadge.swift

```diff
@@ -60,14 +60,29 @@ public struct BPKBadge: View {
         return result
     }
     
+    public func createBadgeIconView(icon: BPKIcon?) -> BPKIconView? {
+        guard let badgeIcon = icon else {
+            switch style {
+            case .normal, .strong, .inverse, .outline, .brand:
+                return nil
```

Comment:
In this case I would use 'default' to return nil. As only the 3 'status' badges receive icons